git/git_intro.qmd

```diff
@@ -1,8 +1,16 @@
 # Introduction to Git
 
-## What is git
+## What is Git
 
-## What should git be used for
+Git is nowadays the most widely used distributed version control system, especially in software development. By opposition to centralized version control systems, with Git, the code source, including its full history, is mirrored on every developer's computer.
+
+Git is the most popular tool, even though it might not be the most user-friendly one. It has a lot of options/commands and a specific jargon. Fortunately there are many "Git cheat sheets" (such as https://education.github.com/git-cheat-sheet-education.pdf).
+
+## What should Git be used for
+
+In software development, Git is mostly used for version control of code. In our bioinformatics projects, we can also track our report files, environment files, and other small files.
+
+Git should NOT be used for storing data, particularly large data. Sensitive data (passwords, usernames, API keys...) should not be put in a Git repository, because they can be then exposed to the world. One can go back into the file history and extract data, so you should remove any problematic files from the Git history as well.
```

Comment:
```suggestion
Git should NOT be used for storing data, particularly large data. Sensitive data (passwords, usernames, API keys...) should not be put in a Git repository, because they can be then exposed to the world. If one commits sensitive data by mistake, one can go back into the git history and remove it, but it is not a simple task.
```
Sorry, normally I wouldn't reword, but I thought this should be emphasized more. Your call.